src/odemis/acq/stream/_projection.py

```diff
@@ -927,6 +927,8 @@ def _updateImage(self):
             data = self._computeSpec()
             if data is not None:
                 self.image.value = self._project2RGB(data)
+            else:
```

Comment:
Some of these changes might conflict with my PR... or not. So let's keep them as-is, and we'll see when my PR is merged.